queries.sql

```diff
@@ -64,4 +64,25 @@ postgres-# FROM animals GROUP BY species;
 -- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
 postgres=# SELECT
 postgres-# species,
-postgres-# AVG(escape_attempts) "Average attempts of escape" FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+postgres-# AVG(escape_attempts) "Average attempts of escape" FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+-- Third task
+
+postgres=# INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34), ('Jennifer Orwell', 19), ('Bob', 45), ('Melody Pond', 77), ('Dean Winchester', 14), ('Jodie Whittaker', 38);
+postgres=# INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
+postgres=# UPDATE animals SET species_id = species.id FROM species WHERE species.name = 'Digimon' AND animals.name LIKE '%mon'; UPDATE animals SET species_id = species.id FROM species WHERE animals.name NOT LIKE '%mon' AND species.name = 'Pokemon';
+postgres=# UPDATE animals SET owner_id = owners.id FROM owners WHERE owners.full_name = 'Sam Smith' AND animals.name = 'Agumon';
+postgres=# UPDATE animals SET owner_id = owners.id FROM owners WHERE owners.full_name = 'Jennifer Orwell' AND (animals.name = 'Gabumon' OR animals.name = 'Pikachu');
+postgres=# UPDATE animals SET owner_id = owners.id FROM owners WHERE owners.full_name = 'Bob' AND (animals.name = 'Devimon' OR animals.name = 'Plantmon');
+postgres=# UPDATE animals SET owner_id = owners.id FROM owners WHERE owners.full_name = 'Melody Pond' AND (animals.name = 'Charmander' OR animals.name = 'Squirtle' OR animals.name = 'Blossom');
+postgres=# UPDATE animals SET owner_id = owners.id FROM owners WHERE owners.full_name = 'Dean Winchester' AND (animals.name = 'Angemon' OR animals.name = 'Boarmon');
```

Comment:
- [ ] You were instructed to add the following data (from line 71 - 78) inside your `data.sql` instead of the queries.sql. Kindly make this 👍🏻 

![10](https://user-images.githubusercontent.com/79058364/184044323-d392f2fc-0471-46cf-a1c8-f9c6838df622.png)

Reply:
Yes, I did add all. I just did the screenshot together.

Reply:
And I was just to add this data, not to screenshot it.

Reply:
Please, Add also the screenshot!!